plugins/tiddlywiki/katex/wrapper.js

```diff
@@ -16,6 +16,19 @@ var katex = require("$:/plugins/tiddlywiki/katex/katex.min.js"),
     chemParse = require("$:/plugins/tiddlywiki/katex/mhchem.min.js"),
 	Widget = require("$:/core/modules/widgets/widget.js").widget;
 
+katex.macros = {};
+katex.updateMacros = function() {
+	var tid, macro, cmd;
+	const regex = /#\d/g; // Remove the arguments like #1#2
+	for (const t of $tw.wiki.getTiddlersWithTag("$:/tags/katex/macro")) {
```

Comment:
In TW core we use ES5 javascript only. So you need to use `var` instead of `const` and the `for of` loop was introduced in ES6. So you need to use the "old way"

```
var tiddlers = $tw.wiki.getTiddlersWithTag("$:/tags/katex/macro") ;
for (var i=0; i < tiddlers.length; i++) {
//your code changed a bit
}
```

Reply:
Thanks! It's fixed.